client/src/components/PhotoItem/PhotoItem.jsx

```diff
@@ -1,12 +1,35 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+
+import classes from './PhotoItem.module.css';
 
 function PhotoItem (props) {
   const { photo } = props;
   const { id, thumbnailUrl, imageUrl, description } = photo;
 
+  const handleClick = (event) => {
+    console.log('it works!')
+  };
+
   return (
-    <img src={thumbnailUrl} alt={description} photo-id={id} />
+    <button type="button" onClick={handleClick} className={classes.photoitem}>
+      <img
+        src={thumbnailUrl}
+        alt={description}
+        photo-id={id}
+        className={
```

Comment:
the logic for the class is probably best written outside the return statement. assign the logic to a const variable and then apply the class with said variable.

Reply:
maybe a switch

Reply:
Logic has been moved into a function.